### ISSUE TITLE
in MA branch - scroll wheel bug

### ISSUE THREAD

**Issue by arobie** [kristinbranson/APT#376]
One thing I've noticed in MA with the scroll wheel to zoom is 'active' when the mouse is over the label frames table depending on the rotation of the animal. So if I'm trying to scroll through the list of labeled frames the zoom of the fly changes instead. I can change the 'focus' of the scroll wheel by clicking on the scroll bars. It's just really unexpected when it happens.

**Comment by allenleetc** [comment]
Yes in the past I've also been surprised sometimes by unexpected 'scrollwheel focus'. But oddly I haven't been able to reproduce your situation yet in the **multianimal** branch. I'm using a withTrx project, running MATLAB 2020b or 2021b on Win10 and Ubuntu. If I scroll the scrollwheel over the main axis, image zoom occurs; if over the Labeled Frames table, that table scrolls; similarly for the Targets table.

What MATLAB version and platform are you using? Are you able to deterministically reproduce this with reproduction steps?
